client/src/components/NewVersionAvailable.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Dialog from './Dialog';
+import css from './NewVersionAvailable.css';
+
+const NewVersionAvailable = ({ newVersionAvailable }) => (
+  <Dialog
+    onClose={() => { window.location.href = '.'; }}
+    title="En ny versjon av applikasjonen er tilgjengelig"
+    visible={newVersionAvailable}
+  >
+    <p>
+      <a href="." className={css.reloadPageLink}>
+        Klikk her for å laste inn siden på nytt.
```

Comment:
There is no way to continue using the application without reloading the page. Clicking on the backdrop or clicking on the link will reload the page. Having the whole dialog clickable seems like a weird behaviour.